sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/sweep/sampling_algorithm.py

```diff
@@ -58,6 +72,17 @@ def __init__(
         seed: int = None,
         logbase: Union[float, str] = None,
     ) -> None:
+        """Random Sampling Algorithm.
+
+        :param rule: The specific type of random algorithm. Possible values include: "random",
```

Comment:
Are there other potential values, or is this list comprehensive? If the latter, should we reword it to "Acceptable values are:"?